mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -1263,10 +1272,9 @@ func expandBackupRetentionSettings(configured interface{}) *sqladmin.BackupReten
 
 func expandActiveDirectoryConfig(configured interface{}) *sqladmin.SqlActiveDirectoryConfig {
 	l := configured.([]interface{})
-	if len(l) == 0 {
+	if len(l) == 0 || l[0] == nil {
 		return nil
 	}
```

Comment:
```suggestion
	if len(l) == 0 {
		return nil
	}
	if l[0] == nil {
		transformed := make(map[string]interface{})
		return transformed
	}
```
If we decide move forward with this change, hopefully this should send the empty block `active_directory_config{}`

Reply:
Will keep this for later use if we end up making this change in a future PR. Thanks!